desktop/apps/categories2/components/AppContainer.js

```diff
@@ -1,8 +1,45 @@
+// Testing testing
+const data = require('../data/data.json')
+
 import React from 'react'
+import Intro from './Intro'
+import Sidebar from './Sidebar'
+import Content from './Content'
+import GeneFamilies from './GeneFamilies'
 
 class AppContainer extends React.Component {
+  constructor(props){
+    super(props)
+    this.contentStyles = this.contentStyles.bind(this)
```

Comment:
You're right. Totally not necessary until those functions start wanting to use props and state.